docs/developer/guidelines/dashboards.md

```diff
@@ -1,3 +1,158 @@
 # Dashboards
 
 -----
+
+Datadog dashboards enable you to efficiently monitor your infrastructure and integrations
+ by displaying and tracking key metrics.
+ 
+## Integration Preset Dashboards
+
+If you would like to create a default dashboard for an integration, follow the guidelines in the [Best Practices](./dashboards.md#best-practices) section.
+
+### Exporting a dashboard payload
+When you've created a dashboard in the Datadog UI, you can export the dashboard payload to be included in its integration's assets directory.
+
+Run the following command to [export the dashboard](../ddev/cli.md#export):
+
+```cli
+ddev meta dash export <URL_OF_DASHBOARD> <INTEGRATION>
+```
+
+!!! tip
+    If the dashboard is for a contributor-maintained integration in the `integration-extras` repo, run the command with the `--extras` or `-e` flag.
+
+The command will add the dashboard definition to the `manifest.json` file of the integration. 
+The dashboard JSON payload will be available in `/assets/dashboards/<INTEGRATION>_overview.json`.
+
+Commit the changes and create a pull request.
+
+## Best Practices
+
+![Mongo dashboard](https://raw.githubusercontent.com/DataDog/integrations-core/master/docs/developer/assets/images/mongo_dashboard.png)
+
+1. When creating a new dashboard, select the Screenboard type.
+
+    !!! information
+        Integration preset (or out of the box) dashboards are typically screenboards because they allow images, widgets, and labels. 
+        Learn more about Datadog dashboards and see the differences between [screenboards and timeboards](https://docs.datadoghq.com/dashboards/#screenboard-vs-timeboard). 
+
+1. Research the metrics supported by the integration and consider grouping them in relevant categories. 
+Important metrics that are key to the performance and overview of the integration should be at the top.
+
+1. Ensure good design by keeping labels and widgets aligned with consistent spacing and reasonable size. 
```

Comment:
Maybe suggest checking the size with the TV mode as in the best practice document

> Make the whole dashboard fit on your horizontal screen (check it with the TV mode).